stack/schema.go

```diff
@@ -47,3 +47,8 @@ type Services struct {
 	Functions map[string]Function `yaml:"functions,omitempty"`
 	Provider  Provider            `yaml:"provider,omitempty"`
 }
+
+type LanguageTemplate struct {
```

Comment:
Comment:

// LanguageTemplate read from template.yml within root of a language template folder